### DISCUSSION TITLE
fill shape with image

### DISCUSSION THREAD

**Discussion by gwenfriedman** [img-mapper/img-mapper#30] [Q&A] [answered]
Hi! I'm trying to use this library for a project and I'm wondering if it's possible to fill the shape on the background image with an image instead of a fill color. My goal is to have the image appear when you are hovering over it. For example, in your live demo if instead of turning the back wall green instead an image of nature appeared in that shape.

**Comment by NishargShah** [comment, marked as answer]
Hello @gwenfriedman, looks like it is similar to this https://github.com/img-mapper/react-img-mapper/issues/7 issue. It's a good idea although but let me think about it.

**Reply by gwenfriedman** [reply]
I was able to make it work with your example code from #7. Thanks!!
